src/emcpy/plots/tools.py

```diff
@@ -0,0 +1,94 @@
+class ColorbarPreset:
+    """
+    Class that has pre-determined vmin, vmax,
+    and cmap values depending on the variable
+    and evaluation type.
+    """
+    def __init__(self, variable, eval_type):
+        """
+        Class constructor.
+
+        Args:
+            variable : (str) variable of interest
+            eval_type : (str) evaluation type
+        """
+
+        cbar_d = {
+            'brightness temperature': {
+                'diff': {
+                    'vmax': -5,
+                    'vmin': 5,
+                    'cmap': 'coolwarm'
+                },
+                'magnitude': {
+                    'vmax': 320,
+                    'vmin': 200,
+                    'cmap': 'rainbow'
+                }
+            },
+            'specific humidity': {
+                'diff': {
+                    'vmin': 0.005,
+                    'vmax': 0.005,
+                    'cmap': 'coolwarm'
+                },
+                'magnitude': {
+                    'vmin': 0,
+                    'vmax': 0.025,
+                    'cmap': 'YlGnBu'
+                }
+            },
+            'temperature': {
+                'diff': {
+                    'vmax': -5,
+                    'vmin': 5,
+                    'cmap': 'coolwarm'
+                },
+                'magnitude': {
+                    'vmax': 320,
+                    'vmin': 200,
```

Comment:
200K is probably a bit extreme? that's -70C, but I guess I don't have a good grasp for 200hPa temps right now

Reply:
I can bump it to 220K so its roughly -50C. the colorbar can always be set to 'extend' to show higher values as well. That way its a consistent 100K spread